### PR TITLE
Converted HC to make sure RefSeq accessions are not predictions

### PR DESCRIPTION
**Pull request by thomasmaurel**

### CODE REVIEW

**Comment by thomasmaurel** [comment]
@james-monkeyshines: I believe this PR can be closed, because the XrefPrefixes cover this issue and more: https://github.com/Ensembl/ensembl-datacheck/pull/128

File: lib/Bio/EnsEMBL/DataCheck/Checks/PredictedXrefs.pm [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'PredictedXrefs',
+  DESCRIPTION    => 'Check that RefSeqs xrefs are predictions',
```
Comment (james-monkeyshines) [inline lib/Bio/EnsEMBL/DataCheck/Checks/PredictedXrefs.pm]:
```suggestion
  DESCRIPTION    => 'Check that RefSeqs xrefs are not predictions',
```

File: lib/Bio/EnsEMBL/DataCheck/index.json [hunk 2]
```diff
@@ -946,6 +946,16 @@
       "name" : "Population",
       "package_name" : "Bio::EnsEMBL::DataCheck::Checks::Population"
    },
+   "PredictedXrefs" : {
+      "datacheck_type" : "critical",
+      "description" : "Check that RefSeqs xrefs are predictions",
```
Comment (james-monkeyshines) [inline lib/Bio/EnsEMBL/DataCheck/index.json]:
```suggestion
      "description" : "Check that RefSeqs xrefs are not predictions",
```

File: lib/Bio/EnsEMBL/DataCheck/Checks/PredictedXrefs.pm [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  GROUPS         => ['compare_core', 'xref'],
```
Comment (james-monkeyshines) [inline lib/Bio/EnsEMBL/DataCheck/Checks/PredictedXrefs.pm]:
```suggestion
  GROUPS         => ['xref'],
```

File: lib/Bio/EnsEMBL/DataCheck/index.json [hunk 4]
```diff
@@ -946,6 +946,16 @@
...
+      "groups" : [
+         "compare_core",
```
Comment (james-monkeyshines) [inline lib/Bio/EnsEMBL/DataCheck/index.json]:
```suggestion
```

**Comment by thomasmaurel** [comment]
This DC cover check from this DC: https://github.com/Ensembl/ensembl-datacheck/pull/128

**Pull request closed without merging**